Nimble/Utils/NotificationCollector.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  NotificationCollector.swift
+//  Nimble
+//
+//  Created by Brian Gerstle on 12/31/15.
+//  Copyright © 2015 Jeff Hui. All rights reserved.
+//
+
+import Foundation
+
+class NotificationCollector {
```

Comment:
Nitpick: Please be explicit about visibility scope for top-level types (add `internal` keyword).